themes/jb/layouts/partials/header.html

```diff
@@ -96,6 +96,11 @@
                       </div>
                       {{ else if eq .Identifier "home"}}
                       <!-- do nothing, home is done as "navbar-brand" class above outside of "navbar-menu" -->
+                      {{ else if eq .Identifier "live"}}
+                          <a id="livebutton" href="{{ .URL }}" {{ if (hasPrefix .Identifier "ext-") }}target="_blank"{{ end }} class="navbar-item {{ if $active }}active{{ end }}">
```

Comment:
Since we know that this is just pointing to `/live` on the main JB website. I don't think we'll need all the code here. This should be enough:

```suggestion
                          <a id="livebutton" href="{{ .URL }}" class="navbar-item {{ if $active }}active{{ end }}">
```

Reply:
No problem at all :slightly_smiling_face: , that's what these reviews are for :grin: 

Also, looks like you might've removed the wrong part. One sec and I'll comment on what happened :sweat_smile: